service/src/main/java/uk/nhs/adaptors/gp2gp/common/storage/StorageUtils.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.adaptors.gp2gp.common.storage;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class StorageUtils {
+    public int getInputStreamSize(InputStream inputStream) throws IOException {
+        ByteArrayOutputStream os = new ByteArrayOutputStream();
+        int b;
```

Comment:
Please, set some more descriptive names to variables.

Reply:
Is this util needed? I think we can just use string.length, what do you think

Reply:
if it is exactly the same size then why not?

Reply:
See my comment in the class that uses this. We don't need this StorageUtils at all.